pyscitt/setup.py

```diff
@@ -25,7 +25,7 @@
     },
     python_requires=">=3.8",
     install_requires=[
-        "ccf==6.0.0-rc0",
+        "ccf==6.0.1",
```

Comment:
package version should also be increased

Reply:
Addressed in https://github.com/microsoft/scitt-ccf-ledger/pull/298/commits/7fe186fc89fd4f3d28f61b555cf96a1f05acc0b3